docs/modules/sql/pages/mapping-to-a-file-system.adoc

```diff
@@ -226,7 +226,9 @@ SELECT * FROM TABLE(
 
 |[[sharedFileSystem]]`sharedFileSystem`
 |`false`
-|Read shared files only once instead of reading each copy on every member.
+|`false` - each member will read all files it sees in the directory
+
+`true` - assume all members see the same files, they will split the work
```

Comment:
```suggestion
`true`: All members see the same files and split the work.
```

Reply:
I'd like to keep the verb "assume" here, because you can set the `sharedFileSystem` property to true even for non-shared file system, no error will be reported, but only a part of the files will be processed.

Reply:
That's important info, so I've added it to the description.